rfcs/0000-rfc-template.md

```diff
@@ -97,6 +108,7 @@ e.g.:
 ## References
 
 <!-- Insert any links appropriate to this RFC in this section. -->
+EPIC with detailed discussion on addition of these fields - https://github.com/elastic/security-team/issues/7658
```

Comment:
I recommend removing any references to Elastic-only discussions or references. For anyone outside Elastic, these links will just return a 404 error.

If you want to link the two discussions, add a link on the private to the public.

Reply:
ACK